akka-stream/src/main/scala/akka/stream/SslTlsOptions.scala

```diff
@@ -0,0 +1,255 @@
+/**
+ * Copyright (C) 2015-2016 Typesafe Inc. <http://www.typesafe.com>
+ */
+package akka.stream
```

Comment:
I find it weird that we go through all the trouble to have a `javadsl` and a `scaladsl` package to separate the APIs and here we just munge them together in the same namespace—I must've missed what's the rationale for this?